app/src/androidTest/java/org/simple/clinic/editpatient/PatientEditScreenUiTest.kt

```diff
@@ -48,7 +48,7 @@ class PatientEditScreenUiTest {
     patientRepository.save(listOf(patientProfile)).blockingAwait()
 
     activityTestRule.runOnUiThread {
-      activityTestRule.activity.screenRouter.clearHistoryAndPush(PatientEditScreen.KEY(patientProfile.patient.uuid), RouterDirection.FORWARD)
+      activityTestRule.activity.screenRouter.clearHistoryAndPush(PatientEditScreen.(::PatientEditScreenKey)(patientProfile.patient.uuid), RouterDirection.FORWARD)
```

Comment:
This is such a weird syntax

Reply:
![fixparrot](https://user-images.githubusercontent.com/2387680/52266528-a4324a00-295c-11e9-80b4-0320e94120ee.gif)